### DISCUSSION TITLE
[555000]GOAT OF DUTY 本体

### DISCUSSION THREAD

**Discussion by hulinhao233** [pjy612/SteamManifestCache#193] [Share]
GOAT OF DUTY
appid:555000
keys:
```
555001;921094082df34744b48b0a6bee4226028cb24548860ca1aedd4a49b57cc6db65"
```
内测CODE:6BF6E0328E98B923
Email:2424344790@qq.com
[555000.zip](https://github.com/pjy612/SteamManifestCache/files/13784739/555000.zip)

**Comment by Malik506-dev** [comment]
is this Call Of Duty ??
